src/core.jsx

```diff
@@ -91,7 +92,13 @@ export default (
 
     render() {
       if (!this.isLoaded()) {
-        return <LoadingIndicator {...this.state.props} />
+        if (LoadingIndicator) {
```

Comment:
Initially @mpolichette used a different property to avoid breaking changes. But i think breaking changes aren't a real problem considering that the lib is quite young and this isn't really a breaking change because you can continue to use the lib as before.